From 56c904f8abdf7416e62dbfaafa02f318fa9a6841 Mon Sep 17 00:00:00 2001
From: Victor Gu <xigu@marvell.com>
Date: Thu, 18 May 2017 10:07:35 +0800
Subject: [PATCH 1007/1345] fix: dts: a3700: add comphy in PCIe node for
 community board

commit  2d5e1866ca04f91872d99c6c71ce414bcc91e2d4 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

The PCIe comphy should be initialized either during cold boot
and PM resume if PCIe is enabled.
The missing of the comphy in community PCIe node will lead to issues
such as suspend to RAM failure during resume. It is because the PCIe
will not work if the PCIe comphy is not initialized, and even worse the
CPU will get stuck when access PCIe controller register if the PCIe
comphy is not initialized.
This patch adds the comphy to be referenced by PCIe node for
community board to fix this issue.
JIRA A3700-1188.

Change-Id: Ic9b16c23e93406a2754dd3e0f8d4125f638ee310
Signed-off-by: Victor Gu <xigu@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/39609
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Evan Wang <xswang@marvell.com>
Reviewed-by: Terry Zhou <bjzhou@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/39671
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/marvell/armada-3720-community.dts     |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-3720-community.dts b/arch/arm64/boot/dts/marvell/armada-3720-community.dts
index 7444d82..34a2b5c 100644
--- a/arch/arm64/boot/dts/marvell/armada-3720-community.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-community.dts
@@ -208,5 +208,7 @@
 
 &pcie0 {
 	status = "okay";
+	phys = <&a3700_comphy 0 COMPHY_PCIE0>;
+	phy-names = "comphy";
 	reset-gpios = <&gpio_sb 3 GPIO_ACTIVE_HIGH>;
 };
-- 
1.7.9.5

